vendor/BUILD

```diff
@@ -17120,3 +17110,13 @@ go_library(
         "//vendor:k8s.io/kube-aggregator/pkg/apis/apiregistration/install",
     ],
 )
+
+go_library(
+    name = "github.com/ClusterHQ/flocker-go",
```

Comment:
I see what is going on. 
The location of this code section has changed. 

There is a deletion in the same file for the same code. 


This is what I get. 
```
[vagrant@k8s-env kubernetes]$ git status
# On branch tpr-unit-tests
nothing to commit, working directory clean
[vagrant@k8s-env kubernetes]$ ./hack/update-bazel.sh
[vagrant@k8s-env kubernetes]$
[vagrant@k8s-env kubernetes]$
[vagrant@k8s-env kubernetes]$ git status
# On branch tpr-unit-tests
nothing to commit, working directory clean
```

update-bazel does not cause any more changes to me.